README.md

```diff
@@ -25,11 +25,14 @@ our repository into your own github account, and run:
 ```bash
 git clone https://github.com/%your_account%/mozdownload.git
 cd mozdownload
-python setup.py develop
+virtualenv venv # configures virtual environment in venv folder
+source venv/bin/activate # activates virtual enviroment
```

Comment:
Instead of adding the steps here, I would suggest to improve the text above to:

`Start by installing these, and [creating a virtualenv for the project](https://docs.python-guide.org/dev/virtualenvs/#lower-level-virtualenv)`.

This will give instructions for all the basics you have to know.

Reply:
ok!